pcsx2/GameDatabase.cpp

```diff
@@ -792,8 +792,8 @@ void GameDatabaseSchema::GameEntry::applyGSHardwareFixes(Pcsx2Config::GSOptions&
 				{
 					if (config.TriFilter == TriFiltering::Automatic)
 						config.TriFilter = static_cast<TriFiltering>(value);
-					else if (config.TriFilter == TriFiltering::Off)
-						Console.Warning("[GameDB] Game requires trilinear filtering but it has been force disabled.");
+					else if (config.TriFilter > TriFiltering::Off)
+						Console.Warning("[GameDB] Game doesn't require trilinear filtering but it has been force enabled.");
```

Comment:
That's the worst replacement message I've seen this year, try again :P

Reply:
What do I win? xD

Reply:
You win a free "Hold Merge" label! congratulations :)